examples/developer-tools/src/blocks/index.ts

```diff
@@ -40,12 +45,18 @@ import '../output-generators/fields/label_serializable';
 import '../output-generators/fields/checkbox';
 import '../output-generators/fields/image';
 import '../output-generators/fields/variable';
+import '../output-generators/fields/angle';
+import '../output-generators/fields/colour';
 import '../output-generators/colour';
 
 /* eslint-disable @typescript-eslint/naming-convention
  -- Blockly convention is to use snake_case for block names
 */
 export const registerAllBlocks = function () {
+  // Register the plugin fields before using them in blocks.
```

Comment:
Do they still need to be registered if we're constructing them explicitly?

Reply:
Yeah, they need to be registered to use them in json, and the block preview is generated from the json block definition.